### PR TITLE
Update cs.json

### PR DESCRIPTION
**Pull request by BlueManCZ**
Czech localization updated.

### CODE REVIEW

File: localization/languages/cs.json [hunk 1]
```diff
@@ -187,16 +192,16 @@
...
-        "appMenuHistory": null, //missing translation
-        "appMenuDeveloper": "Vývojář",
+        "appMenuBookmarks": "Záložky",
+        "appMenuHistory": "Historie",
+        "appMenuDeveloper": "Nástroje pro vývojáře",
         "appMenuReloadBrowser": "Znovu načíst prohlížeč",
-        "appMenuInspectBrowser": "Prozkoumat prohlížeč",
-        "appMenuInspectPage": "Prozkoumat stránku",
+        "appMenuInspectBrowser": "Prozkoumat DOM a styly prohlížeče",
+        "appMenuInspectPage": "Prozkoumat DOM a styly stránky",
```
Comment (PalmerAL) [inline localization/languages/cs.json:200]: Could you make sure the meaning of these matches the English version (unless there's a particular reason to alter them)?

**Comment by PalmerAL** [comment]
Everything looks good aside from my one comment!

File: localization/languages/cs.json (hunk 1 above)
Comment (BlueManCZ) [reply localization/languages/cs.json:200]: I think "Inspect DOM and styles of browser" is more understandable for developers than only "Inspect browser". But I can return it to the previous translation, if you don't agree.

Comment (PalmerAL) [reply localization/languages/cs.json:200]: I think "inspect" is probably sufficient; for comparison, Chrome uses "inspect elements", Firefox uses "inspector", and Safari uses "show web inspector", so I'd prefer to return it to the previous translation.

Comment (BlueManCZ) [reply localization/languages/cs.json]: Alright, I returned it to the previous one.

**Comment by PalmerAL** [comment]
Thank you!

**Pull request merged**